pom.xml

```diff
@@ -114,6 +117,7 @@
                     </execution>
                 </executions>
                 <configuration>
+                    <skip>true</skip>
```

Comment:
```suggestion
                  <skip>true</skip>
```

Reply:
i think we should instead indent the next line